enterprise/reporting/reporting.go

```diff
@@ -445,13 +445,17 @@ func transformMetricForPII(metric types.PUReportedMetric, piiColumns []string) t
 			metric.StatusDetail.SampleEvent = []byte(`{}`)
 		} else if col == "sample_response" {
 			metric.StatusDetail.SampleResponse = ""
+		} else if col == "event_name" {
+			metric.StatusDetail.EventName = ""
+		} else if col == "event_type" {
+			metric.StatusDetail.EventType = ""
```

Comment:
[optional] It looks like a good candidate for a `switch`.

Reply:
done